app/elements/login-popup.html

```diff
@@ -90,7 +97,7 @@
                   label="Username" type="text" value="{{username}}"
                   on-keydown="enterAndTryRegister"></paper-input>
               <paper-input required auto-validate error-message="needs a number!"
-                  label="Student number" type="number" value="{{studentnumber}}"
+                  label="Student number" type="number" value="{{studentnumber}}" min=1000000 max=9999999
```

Comment:
Maybe this is wishful thinking, but should we set the max as a variable to keep it easily changable?